test/unit/helpers/database.py

```diff
@@ -0,0 +1,62 @@
+"""Methods to help with testing database interactions."""
+
+from typing import TypeVar, List
+from psycopg2 import sql
+
+from db_wrapper import Client, ConnectionParameters, ModelData
+from .asynchronous import AsyncMock
+
+
+def composed_to_string(seq: sql.Composed) -> str:
+    """Test helper to convert a sql query to a string for comparison.
+
+    Works for queries built with postgres.sql.Composable objects.
+    From https://github.com/psycopg/psycopg2/issues/747#issuecomment-662857306
+    """
+
+    def compose_array(q: sql.Composed) -> List[str]:
+        # print(f'composing {q}')
+        strings: List[str] = []
+
+        for p in q.seq:
```

Comment:
Just a suggestion, something someone who has reviewed my code mentioned to always detail out these to say what they are doing. This will help with code review down the road. Alphabet salad becomes hard to manage down the road.

Reply:
Resolved in [commit a22ce37](https://github.com/cheese-drawer/transaction-service/pull/7/commits/a22ce372fb223e66ce1caeb7a0299276f846ccb5).